h2o-py/h2o/model/extensions/fairness.py

```diff
@@ -191,6 +194,9 @@ def fair_roc_plot(self, frame, protected_columns, reference, favorable_class, fi
         >>> gbm.fair_roc_plot(test, protected_columns, reference, favorable_class)
         """
         import h2o
+        from h2o.utils.shared_utils import can_use_numpy
+        if not can_use_numpy():
+            raise ImportError("numpy is required for fair_roc_plot.")
```

Comment:
Doesn't use `numpy` directly but indirectly so it's not imported in this function but it is required.